client/ayon_core/tools/publisher/widgets/card_view_widgets.py

```diff
@@ -578,6 +580,9 @@ def __init__(self, controller, parent):
             self.sizePolicy().verticalPolicy()
         )
 
+    def mouseDoubleClickEvent(self, event):
+        self.double_clicked.emit()
```

Comment:
Does this now always go to the publish tab if double clicking - even if not double clicking on an instance? I assume it might but I'm not sure if that's the intended behavior. Is it what we want?

Reply:
the approach is great! thanks

Reply:
> It switches to the publish tab if you double click anywhere within the card view widget.

I would not do that. It should happen only if you double click the card itself. There might be other widgets that want to capture double click on the card itself.

Reply:
> I would not do that. It should happen only if you double click the card itself.

I must admit, I kind of agree - similarly if the "enable/disable" slider on the instance was double clicked (and thus the click was 'handled' already it should not trigger the double click behavior either) - kind of like my [notes here](https://github.com/ynput/ayon-core/pull/143#issuecomment-1984724600).

Reply:
> similarly if the "enable/disable" slider on the instance was double clicked (and thus the click was 'handled' already it should not trigger the double click behavior either)

I have just tested the double click on the slider and nothing happened. It was treated as two single click in sequence, so it disabled and then immediately enabled back again.